cheddar/cheddar-server/src/main/java/com/clicktravel/cheddar/server/runtime/config/RuntimeConfiguration.java

```diff
@@ -61,10 +61,11 @@
 
     public static boolean isLocalEnvironment(final Environment environment) {
         for (final String profile : environment.getActiveProfiles()) {
-            if (profile.equalsIgnoreCase(LOCAL_PROFILE)) {
+            if (profile.equalsIgnoreCase(LOCAL_PROFILE) || profile.equalsIgnoreCase(TEST_PROFILE)
```

Comment:
This change is incorrect. This method is used to determine if locally stored application properties (a properties file with prefix "local-") should be used. At present these are used for the "local" profile, which is now deprecated.

We want to establish two new profiles to replace "local", namely "dev" and "test". Of these two, only "dev" should use locally stored properties. So the condition to test (as we are still using the deprecated "local" profile) should be
```java
if (profile.equalsIgnoreCase(LOCAL_PROFILE) || profile.equalsIgnoreCase(DEV_PROFILE))
```
Accordingly, the name of this method should be changed to isDevEnvironment()